### ISSUE TITLE
SSL Issue

### ISSUE THREAD

**Issue by krishnaguragain** [leokhoa/laragon#82]
```
Laragon Full v4.0.11 190224
Apache 2.4.35 win32 VC15
Laragon installed dir: C:/laragon/
Root Dir: D:/
```

After created new project/folder the SSL goes to Not Secure.
Problem fix only after when the windows is fully restarted(not even sign-out or restarted laragon)

**Comment by BoGnY** [comment]
Hi, are you using Chrome or chromium-based browser??
The error is `ERR_SSL_VERSION_INTERFERENCE`??

Because maybe we have the same problem..

**Comment by dimasahmad** [comment]
@BoGnY I have the same problem with you

**Comment by BoGnY** [comment]
[**@dimasahmad**](https://github.com/dimasahmad) commented on [null](https://github.com/leokhoa/laragon/issues/82#issuecomment-489429408 "2019-05-05T14:02:30Z - Replied by Github Reply Comments"):
> @BoGnY I have the same problem with you

sorry i've abandoned laragon.. now i use docker/vagrant virtual machine for develop and test my project.. best off 👍

**Comment by krishnaguragain** [comment]
Seems like this problem has been fixed in latest laragon release 4.0.14
